tasks/main.yml

```diff
@@ -17,6 +22,33 @@
     state: "{{ docker_service_state }}"
     enabled: "{{ docker_service_enabled }}"
 
+- name: Copy Docker daemon.json file - Docker Configuration requirements CIS section 1
+  copy:
+      src: files/daemon.json
+      dest: /etc/docker/daemon.json
+  when: cis | bool 
+
+- name: Copu audit.rule and restart auditd service - Docker Configuration requirements CIS section 2
+  copy:
+      src: files/audit.rules
+      dest: /etc/audit/audit.rules
+  notify: restart auditd
+  when: cis | bool
+
+- name: Copy fips-module.conf - Configure Docker daemon to use FIPS 140-2 cryptographic module
+  copy:
+      src: files/fips-module.conf
+      dest: /etc/systemd/system/docker.service.d/fips-module.conf
+  when: docker_edition == 'ee'
```

Comment:
We need this only
1. docker_edition is 'ee'
2. /proc/sys/crypto/fips_enabled Not equal to 1